maintenance/cleanup.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# -----------------------------------------------------------------------------
+#   Copyright (C) 2017 University of Dundee. All rights reserved.
+#
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License along
+#   with this program; if not, write to the Free Software Foundation, Inc.,
+#   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ------------------------------------------------------------------------------
+
+
```

Comment:
The cleaning/deleting part of this script does not work.

``Error while cleaning the objects: list index out of range``

Although the listings are as expected, e.g.

```
Will delete 277 ROIs on image                     8258 of trainer-1
Will delete rating 5836 on image                         8258 of trainer-1
Will delete link 3993 on image                             8258 of tag 4237 of trainer-1
Will delete link 3462 on image                             8258 of tag 4236 of trainer-1
Will delete 434 ROIs on image                     8281 of trainer-1
Will delete link 3467 on image                             8281 of tag 4236 of trainer-1
Will delete rating 5833 on image                         8281 of trainer-1
No image with name some-name.dv found
Error while cleaning the objects: list index out of range
```

and no deletion took place.

Tested on both eel and outreach.